src/Symfony/Component/Form/Tests/Extension/Core/Type/LazyChoiceTypeTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\ChoiceList\Loader\CallbackChoiceLoader;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Test\TypeTestCase as TestCase;
+use Symfony\Component\Form\ChoiceList\View\ChoiceView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Tests children of choice type using a default choice loader.
+ *
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+class LazyChoiceTypeTest extends TestCase
```

Comment:
Well, there is strictly nothing testing the runtime code of the CountryType there, as the CountryType does not run.
Testing that fixtures classes not used anywhere else are working properly is useless, as nobody will ever use them anyway.

And if the goal is to test the usage of the CallbackChoiceLoader, this should be done in the tests of the choiceType or of the CallbackChoiceLoader (depending of what they test).